src/js/themes/base.d.ts

```diff
@@ -639,6 +639,35 @@ export interface ThemeType {
     extend?: ExtendType;
     maxHeight?: string;
   };
+  fileInput?: {
+    background?: BackgroundType;
+    border?: BorderType;
+    dragOver?: {
+      background?: BackgroundType;
+      border?: BorderType;
+      pad?: PadType;
+    };
+    extend?: ExtendType;
+    hover?: {
+      background?: BackgroundType;
+      border?: BorderType;
+      pad?: PadType;
+    };
+    icons?: {
+      remove?: any;
+    };
+    label?: {
```

Comment:
Missing the `size` prop

Reply:
Changed to use `TextProps & { extend?: ExtendType; };`